sos/report/plugins/ovn_central.py

```diff
@@ -86,6 +95,9 @@ def setup(self):
             'ovn-sbctl get-ssl',
             'ovn-sbctl get-connection',
             'ovn-nbctl list loadbalancer',
+            'ovn-nbctl list Load_Balancer',
```

Comment:
Do both `loadbalancer` and `Load_Balancer` produce different output? Or is this a versioning change and the former `loadbalancer` is/will be deprecated?

Reply:
Yeah table name is defined at https://github.com/ovn-org/ovn/blob/44ea2ec88136f83e7eab9790473025b6c95bdcc0/ovn-nb.xml#L1516 as Load_Balancer. If I try "loadbalancer" on Ubuntu I get the following so i suspect that other distros maybe be aliasing the name for ease of use - I see that bhaley added that feature [1] pretty recently and I'm sure he did it for a good reason. Maybe I should move this to the Ubuntu/Debian class.

# ovn-nbctl list loadbalancer
ovn-nbctl: unknown table "loadbalancer"

[1] https://github.com/sosreport/sos/commit/bcc897bc4

I will fixup those add_copy_spec() parts